nilearn/plotting/data/html/report_body_template.html

```diff
@@ -0,0 +1,104 @@
+<!-- CSS for the report -->
+<link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css" integrity="sha384-nn4HPE8lTHyVtfCBi5yW9d20FjT8BJwUXyWZT9InLYax14RDjBj46LmSztkmNP9w" crossorigin="anonymous">
```

Comment:
right now other browser-based plots in nilearn work offline. how big is pure-min.css, could it be shipped with nilearn?